tests/performance/src/main/java/org/mule/runtime/core/processor/strategy/ProcessingStrategyBenchmark.java

```diff
@@ -0,0 +1,226 @@
+        .transform(emitterPs.onPipeline(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+    Flux.<CoreEvent>create(s -> workQueuePipeline = s, ERROR)
+        .transform(workQueuePs.onPipeline(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+
+    Flux.<CoreEvent>create(s -> directProcessor = s, ERROR)
+        .transform(directPs.onProcessor(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+    Flux.<CoreEvent>create(s -> emitterProcessor = s, ERROR)
+        .transform(emitterPs.onProcessor(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+    Flux.<CoreEvent>create(s -> workQueueProcessor = s, ERROR)
+        .transform(workQueuePs.onProcessor(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+
+    directAllSink =
+        directPs.createSink(flow, publisher -> baseFlux(publisher, directPs.onPipeline(directPs.onProcessor(processor))));
+    emitterAllSink =
+        emitterPs.createSink(flow, publisher -> baseFlux(publisher, emitterPs.onPipeline(emitterPs.onProcessor(processor))));
+    workQueueAllSink =
+        workQueuePs.createSink(flow,
+                               publisher -> baseFlux(publisher, workQueuePs.onPipeline(workQueuePs.onProcessor(processor))));
+
+  }
+
+  private Flux<CoreEvent> baseFlux(Publisher<CoreEvent> publisher,
+                                   Function<? super Flux<CoreEvent>, ? extends Publisher<CoreEvent>> transformFunction) {
+    return Flux.from(publisher)
+        .transform(transformFunction)
+        .doOnNext(event -> ((MonoSink<CoreEvent>) (event.getMessage().getPayload().getValue())).success(event))
+        .onErrorContinue((t, event) -> {
+          ((MonoSink<CoreEvent>) (((CoreEvent) event).getMessage().getPayload().getValue())).error(t);
+        });
+  }
+
+  @Benchmark
+  @Threads(Threads.MAX)
+  public CoreEvent directSink() {
+    return Mono.<CoreEvent>create(resultSink -> {
+      directSink.accept(createEvent(flow, resultSink));
+    }).block();
+  }
+
+  @Benchmark
+  @Threads(Threads.MAX)
+  public CoreEvent emitterSink() {
+    return Mono.<CoreEvent>create(resultSink -> {
+      emitterSink.accept(createEvent(flow, resultSink));
+    }).block();
+  }
+
+  @Benchmark
+  @Threads(Threads.MAX)
+  public CoreEvent workQueueSink() {
+    return Mono.<CoreEvent>create(resultSink -> {
+      workQueueSink.accept(createEvent(flow, resultSink));
+    }).block();
+  }
+
+  @Benchmark
+  @Threads(1)
+  public CoreEvent directPipeline() {
+    return Mono.<CoreEvent>create(resultSink -> {
+      directPipeline.next(createEvent(flow, resultSink));
+    }).block();
+  }
+
+  @Benchmark
+  @Threads(1)
+  public CoreEvent emitterPipeline() {
+    return Mono.<CoreEvent>create(resultSink -> {
+      emitterPipeline.next(createEvent(flow, resultSink));
+    }).block();
+  }
+
+  @Benchmark
+  @Threads(1)
+  public CoreEvent workQueuePipeline() {
+    return Mono.<CoreEvent>create(resultSink -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove useless curly braces around statement ([squid:S1602](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1602))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7657&issues=AWjJeTCLotIXnmsUTyuc&open=AWjJeTCLotIXnmsUTyuc)

[ISSUE_KEY]: AWjJeTCLotIXnmsUTyuc